plugins/modules/autoscaling_group_info.py

```diff
@@ -414,8 +414,17 @@ def find_asgs(conn, module, name=None, tags=None):
                 if elbv2:
                     try:
                         tg_paginator = elbv2.get_paginator("describe_target_groups")
```

Comment:
A couple of things here 

- I think it's worth moving this describe target groups code out into it's own function.  A function in module_utils/elbv2.py would be perfect, but keeping it in here works too.
- The risk with using an `if` block in here is that we end up with code paths that'll only be hit when people start to scale.  This tends to lead to weird bugs and code getting half-fixed.  I'd recommend *always* chunking here, it give us an extra for-loop but the relative overheads are minimal compared to the API calls themselves, and means that the main code changes aren't going to be tested by the integration tests.

Reply:
I didn't move the function out but removed the ```if``` block per your recommendation. Makes sense and thanks for the context and reasoning behind it.